src/test/java/edu/harvard/iq/dataverse/api/AdminIT.java

```diff
@@ -890,7 +889,41 @@ public void testDownloadTmpFile() throws IOException {
                 .body("status", equalTo("ERROR"))
                 .body("message", equalTo("Path must begin with '/tmp' but after normalization was '/etc/passwd'."));
     }
+    @Test
+    public void testLoadISO_639_controlledVocabularyExt() {
+        Response createUser = UtilIT.createRandomUser();
+        String apiToken = UtilIT.getApiTokenFromResponse(createUser);
+        byte[] updatedContent = null;
+        try {
+            updatedContent = Files.readAllBytes(Paths.get("scripts/api/data/metadatablocks/language_ISO_639.tsv"));
```

Comment:
Once this gets loaded in my installation, is there any way to remove it?

Reply:
no longer relevant